spec/services/dos_to_unix_lines_spec.rb

```diff
@@ -0,0 +1,15 @@
+require 'rails_helper'
+
+describe DosToUnixLines do
+	describe "#call" do
+		global = ActiveSupport::TestCase.fixture_path
+		first_path = File.join("#{global}", "/path1")
+		second_path = File.join("#{global}", "/path2")
```

Comment:
Please, don't use variables in the describe block. These should be in let statement:

```
let(:global) { .... }
```

Also `global` is not a good name for this. Something like `fixture_path` makes more sense.